GUI/MainChangeset.cs

```diff
@@ -47,35 +47,20 @@ public void UpdateChangesDialog(List<ModChange> changeset, BackgroundWorker inst
                 }
 
                 CkanModule m = change.Mod;
-                ListViewItem item = new ListViewItem()
+                ChangesListView.Items.Add(new ListViewItem(new string[]
                 {
-                    Text = m.IsMetapackage
+                    m.IsMetapackage
                         ? string.Format(Properties.Resources.MainChangesetMetapackage, m.name, m.version)
                         : Manager.Cache.IsMaybeCachedZip(m)
                             ? string.Format(Properties.Resources.MainChangesetCached, m.name, m.version)
```

Comment:
IMHO the addition of ` (cached)` is a bit misleading when upgrading mods. It can be unclear if the upgrade or the already installed mod is cached.

I could see another field in ModChange setting the string for the first column which doesn't append ` (changed)`. (that means we'd need another resource string, like `Properties.Resources.MainChangesetInstalled`) which doesn't have the `(cached)` addendum).

It could be used for upgrades, replacements and uninstalls.

Reply:
Fair point, though it seems clear to me that the string in parentheses should describe the mod to be installed (which it does not currently). Easy enough to fix by moving this string into `ModChange`.

Reply:
Works too